DFC.App.ContactUs/Extensions/EnumExtensions.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.ComponentModel;
+using System.Globalization;
+using System.Linq;
+using System.Reflection;
+
+namespace DFC.App.ContactUs.Extensions
+{
+    public static class EnumExtensions
+    {
+        public static string GetDescriptionX<T>(this T e)
```

Comment:
Is this still used or legacy?

Reply:
legacy, forgot to remove, will do so now